bin/critterget.py

```diff
@@ -781,6 +770,34 @@ def getCrashCounts(appId,appName):
     return
 
 
+def setDataRegion(sessionKey):
```

Comment:
This module has a mix of both, let's use PEP8.  Feel free to refactor the other functions in a separate change.